src/checkfiles/checkfiles_local.py

```diff
@@ -113,6 +116,8 @@ def main(args):
     parser.add_argument('--md5sum', help='md5sum of the file to be checked.')
     parser.add_argument('--tabular_file_schema_path',
                         help='the relative path to the schema file of the tabular file.')
+    parser.add_argument('--max_tabular_file_errors', type=int,
+                        help='maximum number of errors to be scaned for the tabular file.')
```

Comment:
Should probably just give this a default value instead of importing constant and doing that check in file_validation function.